### DISCUSSION TITLE
k8s安装好之后，登录遇到问题，登录不了

### DISCUSSION THREAD

**Discussion by zhengxu4869** [hhyo/Archery#2646] [Q&A]

**Comment by zhengxu4869** [comment]
这个初始密码是yaml里定义的这个吗
configMap:
  enabled: true
  # admin password
  superuser:
    username: "admin"
    password: "archery123"  # 请尽快修改

输入这个登录就跳这个弹窗，进不去

**Reply by zhengxu4869** [reply]
![image](https://github.com/hhyo/Archery/assets/53900444/f4eb1555-2405-444a-85d1-21e97a028aa4)

archery-0.3.3

**Comment by LeoQuote** [comment]
看日志，截图没用
